backend/databases/sql.py

```
@@ -214,3 +214,23 @@ def get_user_items(username: str):
+    conn = sqlite3.connect(DATABASE_PATH)
+    cursor = conn.cursor()
+    
+    try:
+        # Delete item from database
+        cursor.execute("DELETE FROM items WHERE id = ? AND user_id = ?", (item_id, username))
+        deleted = cursor.rowcount > 0
+        
+        conn.commit()
+        conn.close()
+        
+        return deleted
```

Ruff (0.14.3)

Consider moving this statement to an `else` block

(TRY300)